examples/ui-demo/src/components/configuration/Styling.tsx

```diff
@@ -112,7 +113,7 @@ export function CornerRadiusOptions() {
         <button
           className={`${
             option.className
-          } py-2 flex-1 basis-0 bg-[#EFF4F9] text-fg-accent-brand hover:opacity-80 ${
+          } py-2 flex-1 basis-0 bg-[#EFF4F9] text-[#363FF9] hover:opacity-80 ${
```

Comment:
we should probably copy `fg-accent-brand` into another variable separate from the css variable that could be overridden in the demo app